content/React Query and management of server state - Nicolás Santos.md

```diff
@@ -0,0 +1,59 @@
+# React Query and management of server state
+
+Some days ago the [React Summit Remote Edition](https://remote.reactsummit.com/) gathered tons of people behind the screen with a promise of put together popular speakers around the React Community like [Kent C. Dodds](https://kentcdodds.com/) and [Guillermo Rauch](https://rauchg.com/) among others, but one talk especially caught my attention, *React Query: It’s Time to Break up with your "Global State”!* by [Tanner Linsley](https://twitter.com/tannerlinsley). In it Tanner talked about two pain points that I had frequently with React, such as how to handle asynchronous information in the global state and how to restructure my code to be less repetitive and not force async data to follow synchronous flow.
+In this article, I explain in more detail which is the problem with the global state and how React Query solves it in an efficient-scalable way. 
+
+## The problem with global state
+Libraries like Redux, MobX and even Context API provided by React are based in the Global State pattern, which means that to avoid prop drilling between the components and have one single point to read and write from the components, the entire application exposes a global variable called _global state_.
```

Comment:
What do you think of re-writing it like this?
```
Libraries like Redux, MobX and even Context API provided by React are based in the Global State pattern, which means that to avoid prop drilling and sharing data between the components in different hierarchy levels, and have one single point to read and write from the components(making it easy to store data used across the app), the entire application exposes a global variable called _global state_.
```

Reply:
I like it. Can you update it?